changelog.txt

```diff
@@ -54,7 +54,8 @@ OPENSEADRAGON CHANGELOG
 * Added a static method in OpenSeadragon to get an existing viewer (#2000 @HerCerM)
 * Now ensuring that the new item is already in the navigator when the "add-item" event fires (#2005 @RammasEchor)
 * Added keys to change image in sequence mode (j: previous, k: next) (#2007 @RammasEchor)
-* Fixed a bug where the navigator wouldn't pick up opacity/composite changes made while it is loading (#2018 @crydell) 
+* Fixed a bug where the navigator wouldn't pick up opacity/composite changes made while it is loading (#2018 @crydell)
+* Removed unnecessary touch-action:none and pointer-events:none settings on overlay container DIV element (#2042 @msalsbery)
```

Comment:
I believe these settings were added during the 3.0 cycle so there's no need to mention removing them in the 3.0 changelog; I'll remove this line.

Reply:
Oh yeah, thank you!